eng/templates/stages/release.yml

```diff
@@ -367,7 +362,7 @@ stages:
             echo "  --sdkVersion $(sdkVersion)"
             echo "  --title $title"
             echo "  --body $body"
-            echo "  --targetBranch $target_branch"
+            echo "  --targetBranch main"
```

Comment:
This doesn't look right.  "main" should only be used for preview releases.  The PRs for servicing releases (e.g. 6.0/7.0) should still me made against the servicing branches.

Reply:
Ah, you're right. Too of a hasty change. Will fix